### PR TITLE
nb/update readme with PR template

### PR DESCRIPTION
**Pull request by nekablair**
### Describe your changes
- Add PR Template to README.md
### Issue ticket number and link
- #_Number
- URL
### Checklist before requesting a review
- [X] I have performed a self-review of my code
- [ ] If it is a core feature, I have added thorough tests.
- [X] Do we need to implement analytics? No
- [ ] Will this be part of a product update? If yes, please write one phrase about this update

### CODE REVIEW

**Comment by GregPetropoulos** [comment]
@istoga can you help me understand what defines a `product update`? does this mean a new  release version of the product? And on the `#_Number` is this something we make up or generate somewhere. Each PR is already numbered when merged. I am happy to make changes  and push to your PR on this.

**Comment by GregPetropoulos** [comment]
The url is also generated after you make the pull request. So in other words the ticket number and url really isn't generated until the PR is submitted. I can remove those from your PR too.

**Comment by nekablair** [comment]
I grabbed a template from somewhere, but we do not need to keep that wording for product update.
The number would be the accompanying issue/ticket #, I wasn't sure if we would or wouldn't be using tickets, we do not have to.
And the URL can be removed too.
There were lots of templates, and many were overly complicated, I wanted something super simple, but to give us some sort of format for consistency. I'm still learning, so if you have other suggestions, please keep them coming.

File: docs/team-procedures.md [hunk 1]
```diff
@@ -7,15 +7,13 @@ If you find a bug, check Trello to see if it's already been reported. If not, ad
...
-### Issue ticket number and link
+- Input changes here
+### Issue ticket number
 - #_Number
-- URL
 ### Checklist before requesting a review
 - [ ] I have performed a self-review of my code
 - [ ] If it is a core feature, I have added thorough tests.
 - [ ] Do we need to implement analytics?
-- [ ] Will this be part of a product update? If yes, please write one phrase about this update.
```
Comment (GregPetropoulos) [inline docs/team-procedures.md]: @istoga Replace this with this below:
 
### Describe your changes
- Describe your changes here and the problem you solved.

### Checklist before requesting a review
- [ ] I have performed a self-review of my code
- [ ] If it is a core feature, I have added thorough tests.
- [ ] Do we need to implement analytics?
- [ ] Screenshots/gifs.
- [ ] Does this PR merge into the main branch.
- [ ] Does this PR merge into the develop branch.
- [ ] Does this PR merge into the staging branch.
- [ ] Describe out of scope work.
---

Comment (nekablair) [reply docs/team-procedures.md]: Done

**Comment by nekablair** [comment]
@GregPetropoulos changed wording

**Review by GregPetropoulos — APPROVED** [review]

**Pull request merged**